.gitignore

```diff
@@ -2,4 +2,5 @@
 node_modules
 coverage
 package-lock.json
-yarn.lock
\ No newline at end of file
+yarn.lock
+.idea
```

Comment:
Remove this line please. Try adding it to your global `.gitignore` (e.g. `~/.gitignore`) if you use the Jetbrains IDEs.